packages/agoric-cli/lib/start.js

```diff
@@ -12,9 +12,9 @@ import {
 
 import { makePspawn, getSDKBinaries } from './helpers.js';
 
-const PROVISION_COINS = `100000000${STAKING_DENOM},50000000000${CENTRAL_DENOM},100provisionpass,100sendpacketpass`;
+const PROVISION_COINS = `100000000${STAKING_DENOM},500000000000000${CENTRAL_DENOM},100provisionpass,100sendpacketpass`;
```

Comment:
this affects the faucet that testnet validators use, right? I was going to ask you about this.

Reply:
Actually, just for local chain use.  On-chain is configured in `packages/deployment`, which we use for testnet genesis generation.